### ISSUE TITLE
Tiny Tapeout 6 submission issues [Action Required]

### ISSUE THREAD

**Issue by urish** [alfiero88/tt06-VCII#1]
Congrats on submitting the project to Tiny Tapeout 6! 

I found two small issues that need addressing before we can merge the project into the chip:

1. In the verilog stub, line 19 is not valid verilog code (`VCII-final` is not a valid verilog identifier):
https://github.com/alfiero88/tt06-VCII/blob/312d3ffe4663126fd306cb6a5bf17d8134c23d57/src/project.v#L19

Since this is just a stub, you can simply remove line 19 through 26 and then it will be good. Otherwise, I'm not sure how to use a macro with a "-" in its name in verilog.

2. In the actual layout, for pin ua[2], the wire has to go all the way to the edge. Currently, there's a spacing of 0.235 um between the end of the wire and the edge, which can cause DRC errors when we integrate it into the bigger chip:

<img width="994" alt="image" src="https://github.com/alfiero88/tt06-VCII/assets/892318/de7a0395-065e-400f-a2f0-276e02f64950">

The other pins are good.

cc @cavelcu

**Comment by alfiero88** [comment]
Hi Urish, first of all, thank you for your support.
For the first point, may I change the verilog identifier into VCII_final, or will it be a problem if it doesn't match with the rest of the file names?
By removing linen 19 to 26, would it result in an overall LVS error? Thanks

**Comment by urish** [comment]
> For the first point, may I change the verilog identifier into VCII_final, or will it be a problem if it doesn't match with the rest of the file names?
> By removing linen 19 to 26, would it result in an overall LVS error? Thanks

We don't do LVS for tt06 analog modules, so you don't need to worry about LVS. Both options would probably work.

**Comment by alfiero88** [comment]
ok I've corrected the pin issue and added the underscore in the identifier. of course I get a lvs error but if it is not a problem, I'll commit the updates

**Comment by urish** [comment]
How do you do LVS? I'm wondering which tool accepts `VCII-final` as a valid identifier in verilog

**Comment by alfiero88** [comment]
I launch make clean lvs in the mag directory, where the tiny tapeout tile that contains my design is located. I guess it double checks the verilog module with this mag file i.e. mag/tt_um_alfiero88_VCII.mag. This file contains the pcell VCII-final.mag, which has the - character

**Comment by urish** [comment]
@mattvenn any idea how LVS is happy with broken verilog?

**Comment by mattvenn** [comment]
netgen probably has its own verilog parser. It doesn't do synthesis, it just knows enough to read module instances and wires. I'm sure it's full of bugs too.

**Comment by alfiero88** [comment]
ok, so are the issues solved? should I do anything else before the tapeout? thanks

**Comment by urish** [comment]
Nothing else to do - it's been merged into the shuttle repo. You should be able to see it on the shuttle map in ~30 minutes:
 https://app.tinytapeout.com/projects/594

**Comment by cavelcu** [comment]
Helly Uri, Thanks for your help. I am glad you solved the problem. I do not recognize this project from the ones of my batch.

**Comment by alfiero88** [comment]
> Nothing else to do - it's been merged into the shuttle repo. You should be able to see it on the shuttle map in ~30 minutes: https://app.tinytapeout.com/projects/594

Ok thank you so much!
